concert/app/test/test_callback.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+import pytest
+from botocore.exceptions import ClientError
+from botocore.stub import Stubber
+from src.functions.callback.index import handler, apigateway
+
+post_to_connection_success_response = {"ResponseMetadata": {"HTTPStatusCode": 200}}
+
+table_name = os.environ["API_URL"]
+
+event_body = {"field1": "value1", "field2": "value2"}
+
+
+def get_event(eventSourceARN="arn:aws:sqs:us-east-1:123456789012:callback-123456"):
+    return {
+        "Records": [{"body": json.dumps(event_body), "eventSourceARN": eventSourceARN}]
+    }
+
+
+def test_callback():
+    event = get_event()
+
+    expected_params = {"Data": json.dumps(event_body), "ConnectionId": "123456="}
+
+    with Stubber(apigateway) as stub:
+        stub.add_response(
+            "post_to_connection", post_to_connection_success_response, expected_params
+        )
+        handler(event, None)
```

Comment:
It seems like this test has no assertions. Shouldn't we at least be checking that the expected call was made? I think we can do [`stub.assert_no_pending_responses()`](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/stubber.html#botocore.stub.Stubber.assert_no_pending_responses). Otherwise this test could pass without any call being made to SNS at all.

Reply:
Good point, and good suggestion. Done.